ui/ui-components/pages/properties.tsx

```diff
@@ -115,6 +115,8 @@ export default function Page(props) {
     );
   }
 
+  const readySources = sources.filter((s) => s.state === "ready");
```

Comment:
This change means you can't see the `draft` properties, which you can today. 

For the UIs in general, I think showing the `deleted` properties is OK - we do that already for Groups and Destinations.

Reply:
Not that this is filtering the state on the _sources_, not the properties. Previously we were only getting `ready` sources, but I changed this to also retrieve the deleted ones. Because of this, I added this filtered array to only show sources that are ready in the select box for the form at the bottom of the page to create a new property.

Reply:
Ah ha! In that case, nice!